src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3309,10 +3308,9 @@ public static void Main()
     }
 }";
             CreateCompilation(source).VerifyDiagnostics(
-    // (6,51): error CS0133: The expression being assigned to 'b' must be constant
-    //         const Func<int> a = () => { const int b = a(); return 1; };
-    Diagnostic(ErrorCode.ERR_NotConstantExpression, "a()").WithArguments("b").WithLocation(6, 51)
-                );
+                // (6,51): error CS0133: The expression being assigned to 'b' must be constant
+                //         const Func<int> a = () => { const int b = a(); return 1; };
+                Diagnostic(ErrorCode.ERR_NotConstantExpression, "a()").WithArguments("b").WithLocation(6, 51));
```

Comment:
>); [](http://example.com/codeflow?start=109&length=2)

Please restore original formatting around the placement of the ```);```. This applies to other changes in this file as well.